README.md

```diff
@@ -1,79 +1,86 @@
+
 # BreezewayMod
+
 ## About
-![](breezewaymod_icon.png) 
 
-BreezewayMod is a fork of the Breezeway window decoration which aims to mimic the macOS interface. But with powerups.
-Did I mention that it comes without weird glitches on the rounded corners?
+<img src="breezewaymod_icon.png" style="zoom: 25%;" /> 
```

Comment:
I think I need to adjust the icon to better fit the Readme. It's just too damn big.